las_geoh5/import_las.py

```diff
@@ -300,6 +304,12 @@ def create_or_append_drillhole(
         translator = LASTranslator()
 
     name = translator.retrieve("well", lasfile)
+    if not name:

```

Comment:
It's trivial but those lines are not covered. 
Just a warning catcher might be nice.